pages/login.tsx

```diff
@@ -0,0 +1,23 @@
+﻿import {LoginForm} from "../components/LoginForm/LoginForm";
+import React, {useState} from "react";
+import scss from "../pageStyles/login.module.scss";
+
+
+export default function Login(): JSX.Element {
```

Comment:
Any chance we could call this `LoginPage`? Just makes it clear that it's a page. I think we're better to mark the return type as `NextPage` too.